.github/workflows/docs.yml

```
@@ -0,0 +1,41 @@
+name: Documentation
+
+on:
+  pull_request:
+  push:
+    branches: [main]
```

I just had to fix this in https://github.com/TEOS-10/GSW-Python/pull/110, so let's port it here.

```suggestion
on:
  pull_request:

  push:
    tags:
      - "v*"
```